pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -0,0 +1,38 @@
+// Ajuna Node
```

Comment:
Maybe we can just have an empty test called `always_good_to_test()`, no need to comment the code here